lib/blockchain/chain.js

```diff
@@ -2506,6 +2506,45 @@ class Chain extends AsyncEmitter {
     return state;
   }
 
+  /**
+   * Get signalling statistics for BIP9/versionbits soft fork
+   * @param {ChainEntry} prev - Previous chain entry.
+   * @param {String} id - Deployment id.
+   * @returns {Promise} - Returns JSON object.
+   */
+
+  async getBIP9Stats(prev, deployment) {
+    const bit = deployment.bit;
+    let window = this.network.minerWindow;
```

Comment:
`window` is a keyword in JavaScript, so its not a good idea to use it as a variable name as it will overwrite the browser window

Reply:
Looks like `window` is used elsewhere in`chain.js` for the same purpose:

https://github.com/bcoin-org/bcoin/blob/a9a60eb5e837f1ff0065f7b25ba40f90f64b66d5/lib/blockchain/chain.js#L2389-L2395

Reply:
I guess the `window` is only overwritten in the scope of the function. I don't think its ideal, but it could work since its already in the codebase. I'd lean towards replacing it but would like to hear opinions from other people